test/e2e/ctrcfg_test.go

```diff
@@ -17,9 +17,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-func TestContainerRuntimeConfigPidsLimit(t *testing.T) {
-	runTestWithCtrcfg(t, "pids-limit", `pids_limit = (\S+)`, "12345", &mcfgv1.ContainerRuntimeConfiguration{
-		PidsLimit: 12345,
+func TestContainerRuntimeConfigLogLevel(t *testing.T) {
+	runTestWithCtrcfg(t, "log-level", `log_level = (\S+)`, "debug", &mcfgv1.ContainerRuntimeConfiguration{
+		LogLevel: "debug",
```

Comment:
why aren't we testing PidsLimit anymore?

Reply:
Because to get the default value of pidsLimit we will have to run `crio config` since we no longer have a crio.conf and that wasn't possible in the MCD pod that the tests use. So moved to LogLevel as we have that set in crio.conf.d/00-default and when we update it we can verify it by reading crio.conf.d/01-ctrcfg-logLevel.